issue/views.py

```diff
@@ -0,0 +1,89 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User
```

Comment:
ここでauth.Userをimportすると、[67行目](https://github.com/SIT-DigiCre/groupware/blob/110fc6766fd7523591f0b575141902d190e3c5bf/issue/views.py#L67)で型エラーが発生しました。
(account.User <- auth.Userにしようとしているため)

私の環境だけの問題かもしれないため、確認お願いします。

```suggestion
from account.models import User
```